com.endlessm.apps.Sdk.json.in.in

```diff
@@ -366,6 +366,22 @@
                 }
             ]
         },
+        {
+            "name": "maxwell",
+            "buildsystem": "meson",
+            "builddir": true,
+            "build-options": {
+                "config-opts": [
+                    "--libdir=/usr/lib"
```

Comment:
Specifying `libdir` shouldn't be necessary, or is it?

Reply:
Ideally, yes, unless Meson inherited from the libdir used when building Python, which some times happens.